README.md

```diff
@@ -25,6 +25,9 @@ Usage: ./check_http_json.rb -u <URI> -e <element> -w <warn> -c <crit>
     -W, --result_warn STRING         Warning if element is [string]. -C is required.
     -U, --result_unknown STRING      Unknown if element is [string]. -C is required.
     -C, --result_crit STRING         Critical if element is [string]. -W is required.
+    -X, --result_warn_regex REGEX    Warning if element matches REGEX. -C is required.
```

Comment:
`--result_warn_regex` is part of the REGEX result group, so `--result_crit_regex` (aka `-D`) is required - not `-C`.